alacritty/src/event.rs

```diff
@@ -411,8 +413,32 @@ impl<N: Notify + OnResize> Processor<N> {
                 GlutinEvent::RedrawEventsCleared => {
                     *control_flow = ControlFlow::Wait;
 
-                    if event_queue.is_empty() {
-                        return;
+                    let event_queue_empty = event_queue.is_empty();
```

Comment:
Seems like there's no need to extract this as a variable? Since it's just conditionally compiled in, it's not executed twice at runtime anyways.

Reply:
I've updated the mentioned code, however I've kept the variable to reduce the amount of conditional code and made it more linear for reading.